modules/log.cpp

```diff
@@ -268,15 +269,35 @@ CString CLogMod::GetServer()
 
 bool CLogMod::OnLoad(const CString& sArgs, CString& sMessage)
 {
+	CString sMyArgs = sArgs;
+	CString sArg;
 	size_t uIndex = 0;
-	if (sArgs.Token(0).Equals("-sanitize"))
-	{
-		m_bSanitize = true;
-		++uIndex;
-	}
 
-	// Use load parameter as save path
-	m_sLogPath = sArgs.Token(uIndex);
+	bool bHaveLogPath = false;
+
+	do {
+		sArg = sMyArgs.Token(uIndex);
+
+		if (sArg.Equals("-sanitize")) {
+			m_bSanitize = true;
+		} else if (sArg.Equals("-timestamp")) {
+			sArg = sMyArgs.Token(++uIndex);
```

Comment:
Can this error? If a user loads the module without a timestamp argument, i.e. `LoadModule = log -timestamp`, what will happen?